### PR TITLE
Fixed coverage for RequestInfoImpl

### PR DESCRIPTION
**Pull request by mrice32**
Did so by expanding the access log format used in the server integration test.

### CODE REVIEW

**Comment by mattklein123** [comment]
nit: This is great, but it would be a better test if you add a RequestInfoImplTest somewhere and make sure that the getters return what you set. (I realize this test does not exist for anything else, but perhaps we can raise the bar :))

**Comment by mrice32** [comment]
Sounds good.  Yeah, I was just following the pattern of how I assumed the other getters were tested.  I'll move it to a standalone `RequestInfoImpl` test.

**Comment by mrice32** [comment]
Added a standalone `RequestInfoImpl` unit test.  Should I remove the formatting change for the integration test or should we keep that?

**Comment by mattklein123** [comment]
I think keeping the integration test changes is fine.

File: test/common/http/access_log/request_info_impl_test.cc [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  }
+
+private:
+  const MonotonicTime pre_start_;
+  RequestInfoImpl request_info_;
+  const MonotonicTime post_start_;
+};
+
+TEST(RequestInfoImplTest, TimingTest) {
+
```
Comment (mattklein123) [inline test/common/http/access_log/request_info_impl_test.cc]: nit: del newline

File: test/common/http/access_log/request_info_impl_test.cc [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  request_info.bytes_sent_ = bytes_sent;
+  request_info.bytes_received_ = bytes_received;
+
+  EXPECT_EQ(bytes_sent, request_info.bytesSent());
+  EXPECT_EQ(bytes_received, request_info.bytesReceived());
+}
+
+TEST(RequestInfoImplTest, ResponseFlagTest) {
+  std::vector<ResponseFlag> responseFlags = {FailedLocalHealthCheck,
```
Comment (mattklein123) [inline test/common/http/access_log/request_info_impl_test.cc]: nit: `const std::vector<ResponseFlag> response_flags`

**Pull request merged**